--- windows/objman.py	2021-06-23 23:12:31.996011626 +0200
+++ /tmp/speakeasy/speakeasy/windows/objman.py	2021-06-23 16:32:20.978074069 +0200
@@ -501,12 +501,12 @@
     """
     An EPROCESS object used by the Windows kernel to represent a process
     """
-    ldr_entries = []
 
     def __init__(self, emu, pe=None, user_modules=[],
                  name='', path='', cmdline='', base=0, session=0):
         super(Process, self).__init__(emu=emu)
         # TODO: For now just allocate a blank opaque struct for an EPROCESS
+        self.ldr_entries = []
         self.object = self.nt_types.EPROCESS(emu.get_ptr_size())
         self.address = emu.mem_map(self.sizeof(), tag=self.get_mem_tag(), perms=1, base=0xe0000000)
         self.name = name
